extension-manifest-v3/src/background/serp-tracking.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { observe } from '/store/options.js';
+
+import { domainList } from './utils/google.js';
+import { toggleDynamicContentScript } from './utils/scripts.js';
+
+observe('terms', async (terms) => {
```

Comment:
I know we had an option in the design for the Settings page, which would allow a user to switch on/off this feature. However, the current design does not have it. Do we want to add an option, or do we want to check only the `terms`?